src/TreeView/widget/ui/Filters.css

```diff
@@ -0,0 +1,9 @@
+
+
+.gv_filter_dropdown_menu .dijitChecked::before {
+    content: √;
```

Comment:
should be configurable??

Reply:
Disagree. This can be changed in a theme if someone wants to use something else. Plus, adding a symbol to `::before` is not quite stable in Javascript.

Reply:
Actually, this css file does not even affect current rendering of the widget, so maybe I should actually delete it entirely. The final styling should be defined in the theme.